### PR TITLE
Test topology pruning

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

File: tests/test_topology_prune.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+"""
+Tests scikit-learn's binarizer converter.
```
Comment (prabhat00155) [inline tests/test_topology_prune.py]: binarizer converter?

File: tests/test_topology_prune.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    def __init__(self):
+        IdentityTransformer.__init__(self)
+
+
+def dummy_shape_calculator(operator):
+    op_input = operator.inputs[0]
+    N = op_input.type.shape[0]
+    C = op_input.type.shape[1]
+    operator.outputs[0].type = FloatTensorType([N, C])
```
Comment (prabhat00155) [inline tests/test_topology_prune.py]: Do we really need N and C here? We could just write: 
operator.outputs[0].type = FloatTensorType(op_input.type.shape), couldn't we?

**Comment by prabhat00155** [comment]
Why do you have so many commits in the PR, may be clean up your branch?

File: tests/test_topology_prune.py (hunk 2 above)
Comment (xadupre) [reply tests/test_topology_prune.py]: I don't know. I update the master branch on my repo and I create new branches with git checkout -b …

Comment (prabhat00155) [reply tests/test_topology_prune.py]: You fork is not synchronised with master. Look at this: https://stackoverflow.com/questions/41283955/github-keeps-saying-this-branch-is-x-commits-ahead-y-commits-behind

**Pull request merged**